lib/datasets.go

```diff
@@ -235,6 +235,11 @@ func (r *DatasetRequests) Get(p *GetParams, res *GetResult) (err error) {
 	}
 	ctx := context.TODO()
 
+	// Check if the dataset ref uses bad-case characters, show a warning.
+	if _, err := dsref.Parse(p.Path); err == dsref.ErrBadCaseName {
+		log.Error(dsref.ErrBadCaseShouldRename)
```

Comment:
works for now, we should have a long conversation about logging & how we actually want it to work. Another time

Reply:
Another option would be to set a value on the lib.Get result, and have the caller display the warning instead.